### DISCUSSION TITLE
Transfer project (not repo, I mean project) to my organisation

### DISCUSSION THREAD

**Discussion by CRoederTND** [community/community#102758] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Cheers,

I had a repository with a corresponding project (the new type). I transfered the ownership of the repo to my newly created company, but the project is still under my personal account. Also there is no way to link the project in my personal account the the repo in the company account. Is there a way to transfer the ownership of a project? Or any way to link the project?

Thanks in advance!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by JamesParsons** [comment]
I would also like to know the answer to this.
